common/data_refinery_common/models/jobs.py

```diff
@@ -171,6 +171,7 @@ class Meta:
     original_files = models.ManyToManyField('OriginalFile', through='ProcessorJobOriginalFileAssociation')
     datasets = models.ManyToManyField('DataSet', through='ProcessorJobDataSetAssociation')
     no_retry = models.BooleanField(default=False)
+    abort = models.BooleanField(default=False)
```

Comment:
Isn't this the correct place to put `null=True` instead of directly in the migration?

Reply:
Ok, I get it now. We want `null=False` eventually (which is the default so this matches that), but first we gotta set it to True for the migration, then we can go back and reset it back to false.